examples/hybrid/driver.jl

```diff
@@ -82,7 +105,7 @@ include(joinpath("sphere", "baroclinic_wave_utilities.jl"))
 const sponge = false
 
 # Variables required for driver.jl (modify as needed)
-params = BaroclinicWaveParameterSet()
+const params = BaroclinicWaveParameterSet()
```

Comment:
What's the benefit of making this a `const`? It prevents us from looping over different parameter sets, and I don't see how it improves efficiency.

Reply:
We don't have to, I think I added this because I was running into inference issues and I suspected this might be the problem